test/services/claimToAdjust/ClaimToAdjustServiceSpec.scala

```diff
@@ -340,22 +329,21 @@ class ClaimToAdjustServiceSpec extends TestSupport with MockFinancialDetailsConn
       ))
 
       val f = fixture(LocalDate.of(taxYear, 8, 27))
-      val result = f.testClaimToAdjustService.getAmendablePoaViewModel(testUserNino)
+      val futureResult = f.testClaimToAdjustService.getAmendablePoaViewModel(testUserNino)
+
+      val result = futureResult.futureValue
+
+      result.isLeft shouldBe true
+      result.left.exists(_.getMessage == "Failed to retrieve non-crystallised financial details") shouldBe true
```

Comment:
if this can be done by use of next scala test trait: https://www.scalatest.org/user_guide/using_EitherValues
So we can compare to exact value in the Exception like: 

`result.left.value.getMessage shouldBe "Failed to retrieve non-crystallised financial details"`